content/applications/productivity/data_cleaning.rst

```diff
@@ -0,0 +1,209 @@
+of eligible records is shown to the right of the :icon:`oi-arrow-right` :guilabel:`# record(s)`.
+
+Choose whether to :guilabel:`Archive` or :guilabel:`Delete` the merged records in the
+:guilabel:`Duplicate Removal` field.
+
+Next, select a :guilabel:`Merge Mode`:
+
+- :guilabel:`Manual`: requires each duplicate grouping to be manually merged, also enables the
+  :guilabel:`Notify Users` field.
+- :guilabel:`Automatic`: automatically merges duplicate groupings, without notifying users, based on
+  the records with a similarity percentage above the threshold set in the :guilabel:`Similarity
+  Threshold` field.
+
+Enable the :guilabel:`Active` toggle to start capturing duplicates with this rule as soon as it is
+saved.
+
+Lastly, create at least one :guilabel:`Deduplication Rules` by clicking :guilabel:`Add a line`.
+
+- Select a field in the model from the :guilabel:`Unique ID Field` drop-down menu. This field will
+  be referenced for similar records.
+- Select a matching condition in the :guilabel:`Match If` field to apply the duplication rule
+  depending on the text in the :guilabel:`Unique ID Field`:
+
+  - :guilabel:`Exact Match`: the characters in the text match exactly.
+  - :guilabel:`Case/Accent Insensitive Match`: the characters in the text match regardless of casing
+    and language-specific accent differences.
+
+.. note::
+   The rule will not capture duplicates without at least one :guilabel:`Deduplication Rules` set.
+
+With the rule's configuration complete, either close the rule form or :ref:`run the rule manually
+<data-cleaning/run-deduplication-rule>` to instantly capture duplicate records.
+
+.. _data-cleaning/run-deduplication-rule:
+
+Manually run a deduplication rule
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To manually run a specific deduplication rule at any time, navigate to :menuselection:`Data Cleaning
+--> Configuration --> Rules: Deduplication` and select the rule to run.
+
+Then, on the rule form, select the :guilabel:`Deduplicate` button on the top-left. Upon doing so,
+the :icon:`fa-clone` :guilabel:`Duplicates` smart button displays the number of duplicates captured.
+
+Click on the :icon:`fa-clone` :guilabel:`Duplicates` smart button to manage these records.
+
+Recycle records
+===============
+
+In the *Recycle Records* dashboard (:menuselection:`Data Cleaning --> Recycle Records`), Odoo
+detects records that can be archived or deleted by matching conditions within the records set by the
+recycle records rules.
+
+Field cleaning
+==============
+
+In the *Field Cleaning* dashboard (:menuselection:`Data Cleaning --> Field Cleaning`), Odoo
+suggests formatting changes to data in fields of a record, to follow a convention set by the field
+cleaning rules.
+
+.. _data_cleaning/merge-action-manager:
+
+Merge action manager
+====================
+
+The *Merge Action Manager* (:menuselection:`Data Cleaning --> Configuration --> Merge Action
```

Comment:
```suggestion
The :guilabel:`Merge Action Manager` (:menuselection:`Data Cleaning app --> Configuration --> Merge Action
```
I wasn't able to locate this during my revisions. Might be worth double-checking to ensure the accuracy of this portion.

![Screenshot 2024-06-21 090356](https://github.com/odoo/documentation/assets/73958186/006bc348-b96d-4495-91d7-b3f6146127ba)